File: Study/JAVA/Android/ListViewTest/app/src/main/java/com/example/myapplication/Board.java
package com.example.myapplication;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Board {


    @SerializedName("pid")
    List<Long> pid;

    @SerializedName("title")
    List<String> title;

    @SerializedName("content")
    List<String> content;
}

File: Study/JAVA/Android/app/src/main/java/com/example/testlistview/Board.kt
package com.example.testlistview

import kotlin.properties.Delegates


public class Board {
    lateinit var pid : String
    lateinit var title : String
    lateinit var content : String

    constructor()

    constructor(pid: String, title: String, content: String) {
        this.pid = pid
        this.title = title
        this.content = content
    }



}